client/aenea/alias.py

```diff
@@ -154,7 +155,7 @@ def split(self, text):
         self._update_regex()
         k = 0 # The end of the previous match.
         open_angle_brackets = 0
-
+        
```

Comment:
@nirvdrum My editor sometimes inserts extra whitespace.  It seems like a non-issue to me.